easybuild/easyconfigs/l/libXrandr/libXrandr-1.4.2-intel-2015a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libXrandr'
+version = '1.4.2'
+
+description = "libXrandr - X Resize, Rotate and Reflection extension library"
+homepage = 'http://cgit.freedesktop.org/xorg/lib/libXrandr/'
+
+# The following are automatically calculated - cross fingers
+sources = ['libXrandr-1.4.2.tar.gz']
+source_urls = ['http://xorg.freedesktop.org/archive/individual/lib/','download']
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['.'] # fixme when the compilation is successful
+}
+
+dependencies = [
+    ('xproto', '7.0.27'),
+    ('xextproto', '7.3.0'),
+    ('libXext', '1.3.3'),   
+    ('libXrender', '0.9.8'),
+    ('libX11', '1.6.2', '-Python-2.7.9'),
+    ('randrproto', '1.4.3'),
+    ('renderproto', '0.11.1')
+]
+           
+
+parallel = 1 # this is a conservative choice
+maxparallel = 1 # this is a conservative choice
```

Comment:
I would drop the `maxparallel`. That really doesn't have any use if `parallel` is already set.